packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -181,6 +181,18 @@ async function unpack(prepareOptions: PrepareApplicationStageDirectoryOptions, o
       return
     }
     await executeAppBuilder(["unpack-electron", "--configuration", JSON.stringify([options]), "--output", appOutDir, "--distMacOsAppName", distMacOsAppName])
+    if (packager.config.downloadAlternateFFmpeg) {
+      log.info("downloading and unpacking alternate FFmpeg")
+      const ffmpegOptions = Object.assign({}, options)
+      ffmpegOptions.customFilename = `ffmpeg-v${options.version}-${options.platform}-${options.arch}.zip`
```

Comment:
Looking at the logic above for `const zipFile = `electron-v${options.version}-${platformName}-${options.arch}.zip``, I think we can use `platformName` instead of `options.platform`? That way we're labeling for sure the same download file?

Reply:
the naming logic here mirrors the `unpack-electron` options (since the full `options` object is just passed to the app-builder-bin call), since this case is only handled when we are downloading the default electron distributable (i.e. `dist === null`). The zipFile reference above is referencing something the user can manually override and specify custom platforms for, and we won't run this ffmpeg option in such a case. The user is expected to have the dlls in that electron distributable the way they want them.